search/src/Components/wrapper/reactive_search.js

```diff
@@ -179,6 +179,7 @@ class ReactiveSearchComponent extends Component {
 												</ul>
 												{" in " + item.agcontext__growth_stage_texts + " " + item.agcontext__crop_type}
 											</ResultCard.Description>
+											<div><a title={item.info ? item.info.name : ""} href={esURL + item.dataset_url}>See Dataset</a></div>
```

Comment:
this doesn't work yet, right?

Reply:
Indexing upload_id currently not working, coz previously was indexing dataset url. I'm troubleshooting now.